src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -1259,6 +1323,16 @@ static ucs_status_t uct_ib_mlx5_devx_md_open(struct ibv_device *ibv_device,
                   uct_ib_device_name(dev));
     }
 
+    if (uct_ib_mlx5_mkey_by_name_reserve_set(md, cap_2)) {
+        md->flags |= UCT_IB_MLX5_MD_FLAG_MKEY_BY_NAME_RESERVE;
```

Comment:
can we make uct_ib_mlx5_mkey_by_name_reserve_set return void, and set md->flags internally?

Reply:
fixed, also fixed existing similar xgvmi related similarly